alpha/lib/model/kuser.php

```diff
@@ -158,6 +158,11 @@ public function postUpdate(PropelPDO $con = null)
 		if($this->isColumnModified(kuserPeer::STATUS) && $this->getStatus() == KuserStatus::DELETED) {
 			$objectDeleted = true;
 		}
+		
+		if ($this->isCustomDataModified(null, 'is_sso_excluded'))
```

Comment:
you should pass the const for is_sso_excluded as the first arg, the field name is the name var in the customDataModified function